src/components/RequestsTable/RequestsTable.tsx

```diff
@@ -0,0 +1,204 @@
+    return "Request Type";
+  },
+  renderCell: (item) => {
+    return <TableCellLayout truncate>{item.requestType}</TableCellLayout>;
+  },
+});
+
+const SystemSeriesGrade = createTableColumn<RPARequest>({
+  columnId: "systemSeriesGrade",
+  renderHeaderCell: () => {
+    return "PS-SS-GR";
+  },
+  renderCell: (item) => {
+    return (
+      <TableCellLayout truncate>
+        {item.paySystem + "-" + item.series + "-" + item.grade}
+      </TableCellLayout>
+    );
+  },
+});
+
+const OfficeSymbol = createTableColumn<RPARequest>({
+  columnId: "officeSymbol",
+  renderHeaderCell: () => {
+    return "Office Symbol";
+  },
+  renderCell: (item) => {
+    return <TableCellLayout truncate>{item.officeSymbol}</TableCellLayout>;
+  },
+});
+
+const Requestor = createTableColumn<RPARequest>({
+  columnId: "requestor",
+  renderHeaderCell: () => {
+    return "Requestor";
+  },
+  renderCell: (item) => {
+    return (
+      <TableCellLayout
+        truncate
+        media={
+          <Avatar aria-label={item.Author?.Title} name={item.Author?.Title} />
+        }
+      >
+        {item.Author?.Title}
+      </TableCellLayout>
+    );
+  },
+});
+
+const CurrentStage = createTableColumn<RPARequest>({
+  columnId: "currentStage",
+  renderHeaderCell: () => {
+    return "Current Stage";
+  },
+  renderCell: (item) => {
+    return <TableCellLayout truncate>{item.stage}</TableCellLayout>;
+  },
+});
+
+const CreatedDate = createTableColumn<RPARequest>({
+  columnId: "createdDate",
+  renderHeaderCell: () => {
+    return "Created Date";
+  },
+  renderCell: (item) => {
+    return (
+      <TableCellLayout truncate>
+        {item.Created?.toLocaleDateString()}
+      </TableCellLayout>
+    );
+  },
+});
+
+const columns: TableColumnDefinition<RPARequest>[] = [
+  PositionTitle,
+  RequestType,
+  SystemSeriesGrade,
+  OfficeSymbol,
+  Requestor,
+  CurrentStage,
+  CreatedDate,
+];
+
+const RequestsTable = () => {
+  const [page, setPage] = useState(0);
+  const pagedItems = usePagedRequests(page);
+
+  return (
+    <>
+      <DataGrid
+        items={pagedItems.data?.results || []}
+        columns={columns}
+        getRowId={(item) => item.Id}
+        resizableColumns
+        columnSizingOptions={columnSizingOptions}
+      >
+        <DataGridHeader>
+          <DataGridRow>
+            {({ renderHeaderCell }) => (
+              <DataGridHeaderCell>{renderHeaderCell()}</DataGridHeaderCell>
+            )}
+          </DataGridRow>
+        </DataGridHeader>
+        <DataGridBody<RPARequest>>
+          {({ item, rowId }) => (
+            <DataGridRow<RPARequest> key={rowId}>
+              {({ renderCell }) => (
+                <DataGridCell>{renderCell(item)}</DataGridCell>
+              )}
+            </DataGridRow>
+          )}
+        </DataGridBody>
+      </DataGrid>
+      <div style={{ display: "flex", justifyContent: "space-between" }}>
+        <Button
+          appearance="primary"
+          disabled={page <= 0}
```

Comment:
Is it possible for this to be less than 0?
```suggestion
          disabled={page === 0}
```